var/spack/repos/builtin/packages/libglvnd/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libglvnd(AutotoolsPackage):
+    """The GL Vendor-Neutral Dispatch library
+
+    libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API
+    calls between multiple vendors. It allows multiple drivers from different
+    vendors to coexist on the same filesystem, and determines which vendor to
+    dispatch each API call to at runtime.
+
+    Both GLX and EGL are supported, in any combination with OpenGL and OpenGL
+    ES."""
+
+    homepage = "https://github.com/NVIDIA/libglvnd"
+    url      = "https://github.com/NVIDIA/libglvnd/releases/download/v1.1.1/libglvnd-1.1.1.tar.gz"
+    git      = "https://github.com/NVIDIA/libglvnd.git"
+
+    version('master', branch='master')
+
+    version('1.1.1', sha256='71918ed1261e4eece18c0b74b50dc62c0237b8d526e83277ef078554544720b9')
+    version('1.1.0', sha256='f5a74598e769d55d652c464cb6507437dac5c2d513f16c6ddf3a1bec655a1824')
+
```

Comment:
glvnd is also linux specific, although I'm not quite sure the best way to convey that, i.e. conflict when *not* linux or cray (since cray is linux).

@adamjstewart ?

Reply:
I usually do something like:
```python
conflicts('platform=darwin', msg='glvnd is linux specific')
```
Unfortunately there's no "not" in the spec syntax atm.

Reply:
It seems like the only platforms right now are `bgq`, `cray`, `darwin`, and `linux`.  So we'd want to conflict for both `bgq` and `darwin`.